Tests/Functional/DSL/Aggregation/ReverseNestedAggregationTest.php

```diff
@@ -0,0 +1,270 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\ElasticsearchBundle\Tests\Functional\DSL\Aggregation;
+
+use ONGR\ElasticsearchBundle\DSL\Aggregation\NestedAggregation;
+use ONGR\ElasticsearchBundle\DSL\Aggregation\ReverseNestedAggregation;
+
+use ONGR\ElasticsearchBundle\DSL\Aggregation\TermsAggregation;
+use ONGR\ElasticsearchBundle\DSL\Aggregation\FilterAggregation;
+use ONGR\ElasticsearchBundle\DSL\Filter\TermFilter;
+use ONGR\ElasticsearchBundle\ORM\Repository;
+use ONGR\ElasticsearchBundle\Test\ElasticsearchTestCase;
+use ONGR\ElasticsearchBundle\Test\TestHelperTrait;
+
+class ReverseNestedAggregationTest extends ElasticsearchTestCase
```

Comment:
`ElasticsearchTestCase` is deprecated. Please use [AbstractElasticsearchTestCase](https://github.com/ongr-io/ElasticsearchBundle/blob/master/Test/AbstractElasticsearchTestCase.php)